src/net/sourceforge/kolmafia/swingui/FaxRequestFrame.java

```diff
@@ -54,7 +54,7 @@ public FaxRequestFrame() {
       panel.add(botPanel);
       this.selectorPanel.addPanel(bot.getName(), panel);
     }
-    this.selectorPanel.setSelectedIndex(0);
+    this.selectorPanel.setSelectedIndex(Preferences.getInteger("faxbots"));
```

Comment:
I'm not finding this preference in defaults.  Please add it.  It might be worth discussing whether it should be a global or user preference.  If I prefer a faxbot for performance reasons then my preference won't change when I use a different character.

Reply:
Added as a user faxbot, though it could be added as a global